CMSgov/design-system
# [WNMGDS-507] Fix cannot read property 'checked' of null

### Changed
- Fix the issue of `Cannot read property 'checked' of null` after a checked parent uncontrolled radio button (with children radio buttons) gets deselected. After that, any subsequent selecting of the children radio buttons will result in the error msg. 

### How to test
- Run the design system site locally and check that Choice component examples works. Specifically check that  `Radio buttons with multi-level children` works as intended and inspect dev tools to ensure that it does not throw errors.

### Fixed
Fixes #717
